packages/unhead/src/plugins/safe.ts

```
@@ -111,122 +82,73 @@ function acceptDataAttrs(value: Record<string, string>, allowId = true) {
   )
 }
 
+function pickWhitelisted(prev: Record<string, any>, whitelist: Set<string>): Record<string, any> {
+  const o: Record<string, any> = {}
+  for (const k of whitelist) {
+    if (prev[k]) o[k] = prev[k]
```

ESLint

[error] Expect newline after if

(antfu/if-newline)